html2haml.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 1.9.2'
 
-  gem.add_dependency 'nokogiri', '~> 1.6.0'
+  gem.add_dependency 'nokogiri', RUBY_VERSION < '2.1' ? '~> 1.6.0' : '>= 1.7'
```

Comment:
Thank you for comment 👍 
When you `specify '~> 1.6'`, target version is `2.0 > version >= 1.6`.My intension is not to include Nokogiri 1.7 or above(Nokogiri required Ruby 2.1 or above since 1.7.0).